src/test/java/com/wikia/webdriver/testcases/discussions/ZeroErrorStateTests.java

```diff
@@ -0,0 +1,230 @@
+  public void staffUserOnDesktopSeeProperMessageWhenUploadsEmptyReportedPostsPage() {
+    userSeeProperMessageWhenUploadsEmptyReportedPostsPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnDesktopSeeProperMessageWhenUploadsEmptyPostsListPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void staffUserOnDesktopSeeProperMessageWhenUploadsEmptyPostsListPage() {
+    userSeeProperMessageWhenUploadsEmptyPostsListPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void staffUserOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage() {
+    userOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnDesktopSeeProperMessageWhenUploadsEmptyPostDetailsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+  public void staffUserOnDesktopSeeProperMessageWhenUploadsEmptyPostDetailsPage() {
+    userOnDesktopSeeProperMessageWhenUploadsEmptyPostDetailsPage();
+  }
+
+  /**
+   * STAFF USERS ON MOBILE SECTION
+   */
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsEmptyReportedPostsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsEmptyReportedPostsPage() {
+    userSeeProperMessageWhenUploadsEmptyReportedPostsPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostsListPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostsListPage() {
+    userSeeProperMessageWhenUploadsEmptyPostsListPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsNonExistingUserPostsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsNonExistingUserPostsPage() {
+    userOnMobileSeeProperMessageWhenUploadsNonExistingUserPostsPage();
+  }
+
+  @Test(groups = "discussions-staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostDetailsPage")
+  @Execute(asUser = User.STAFF)
+  @InBrowser(
+      browser = Browser.CHROME,
+      emulator = Emulator.GOOGLE_NEXUS_5)
+  public void staffUserOnMobileSeeProperMessageWhenUploadsEmptyPostDetailsPage() {
+    userOnMobileSeeProperMessageWhenUploadsEmptyPostDetailsPage();
+  }
+
+  /**
+   * TESTING METHODS SECTION
+   */
+
+  private void userOnDesktopSeeProperMessageWhenUploadsNonExistingUserPostsPage() {
```

Comment:
I believe that the action performed by that user is "opening" non existing user post page
Not "uploading" non existing user post page

Please update the method and tests names so that it states:
userOnDesktopSeesProperMessageWhenOpensNonExistingUserPostsPage